frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -191,32 +162,37 @@ plans:
         strategy: serial
         pod: node
         steps:
-          - 0: [[backup-schema], [upload-schema-s3]]
+          - default: [[backup-schema]]
```

Comment:
It's packaged in the backup folder for the node instead and uploaded along with the snapshots.